--- daemon/Makefile.am	2022-01-23 08:12:00.000000000 +0100
+++ Makefile.am_new	2022-01-23 08:20:55.000000000 +0100
@@ -137,7 +137,6 @@
 	headtail.c \
 	hexdump.c \
 	hotplug.c \
-	hivex.c \
 	htonl.c \
 	initrd.c \
 	inotify.c \
@@ -214,8 +213,6 @@
 	zerofree.c
 
 guestfsd_LDFLAGS = \
-	-L$(shell $(OCAMLC) -where) \
-	-L$(shell $(OCAMLFIND) query hivex) \
 	-L../common/mlutils \
 	-L../common/mlstdutils \
 	-L../bundled/ocaml-augeas \
@@ -227,7 +224,6 @@
 	$(JANSSON_LIBS) \
 	$(SELINUX_LIBS) \
 	$(AUGEAS_LIBS) \
-	$(HIVEX_LIBS) \
 	$(SD_JOURNAL_LIBS) \
 	$(top_builddir)/gnulib/lib/.libs/libgnu.a \
 	$(LIBSOCKET) \
@@ -261,7 +257,6 @@
 	$(WARN_CFLAGS) $(WERROR_CFLAGS) \
 	$(RPC_CFLAGS) \
 	$(AUGEAS_CFLAGS) \
-	$(HIVEX_CFLAGS) \
 	$(SD_JOURNAL_CFLAGS) \
 	$(JANSSON_CFLAGS) \
 	$(PCRE2_CFLAGS) \
@@ -354,7 +349,7 @@
 XOBJECTS = $(BOBJECTS:.cmo=.cmx)
 
 OCAMLPACKAGES = \
-	-package str,unix,hivex \
+	-package str,unix \
 	-I $(top_builddir)/bundled/ocaml-augeas \
 	-I $(top_builddir)/common/mlstdutils \
 	-I $(top_builddir)/common/mlutils \
@@ -375,7 +370,6 @@
 	-lmlcutils \
 	-lmlstdutils \
 	-lmlaugeas \
-	-lmlhivex \
 	-lcamlstr \
 	-lunix \
 	-l$(CAMLRUN) -ldl -lm
@@ -453,7 +447,7 @@
 	$(top_builddir)/ocaml-link.sh
 daemon_utils_tests_LINK = \
 	$(top_builddir)/ocaml-link.sh \
-	  -cclib '-lutils -lgnu $(HIVEX_LIBS)' \
+	  -cclib '-lutils \
 	  -- \
 	  $(OCAMLFIND) $(BEST) $(OCAMLFLAGS) $(OCAMLLINKFLAGS) \
 	  $(OCAMLPACKAGES) \
