--- configure.ac	2021-09-07 18:26:15.000000000 +0200
+++ configure.ac_new	2022-01-25 23:33:23.000000000 +0100
@@ -138,33 +138,16 @@
 HEADING([Checking for other libraries])
 m4_include([m4/guestfs-misc-libraries.m4])
 
-dnl Check for language bindings.
-HEADING([Checking for OCaml])
-m4_include([m4/guestfs-ocaml.m4])
-HEADING([Checking for Perl])
-m4_include([m4/guestfs-perl.m4])
-HEADING([Checking for Python])
-m4_include([m4/guestfs-python.m4])
-HEADING([Checking for Ruby])
-m4_include([m4/guestfs-ruby.m4])
-HEADING([Checking for Java])
-m4_include([m4/guestfs-java.m4])
-HEADING([Checking for Haskell])
-m4_include([m4/guestfs-haskell.m4])
-HEADING([Checking for PHP])
-m4_include([m4/guestfs-php.m4])
-HEADING([Checking for Erlang])
-m4_include([m4/guestfs-erlang.m4])
-HEADING([Checking for Lua])
-m4_include([m4/guestfs-lua.m4])
-HEADING([Checking for Go])
-m4_include([m4/guestfs-golang.m4])
-HEADING([Checking for GObject Introspection])
-m4_include([m4/guestfs-gobject.m4])
-HEADING([Checking for Rust])
-m4_include([m4/guestfs-rust.m4])
-HEADING([Checking for Vala])
-VAPIGEN_CHECK
+AM_CONDITIONAL([HAVE_TOOLS], [false])
+AM_CONDITIONAL([HAVE_OCAMLOPT], [false])
+AM_CONDITIONAL([HAVE_OCAML_PKG_GETTEXT], [false])
+AM_CONDITIONAL([HAVE_OCAML], [false])
+AM_CONDITIONAL([HAVE_BYTES_COMPAT_ML], [false])
+AM_CONDITIONAL([HAVE_OCAML_PKG_OUNIT], [false])
+AM_CONDITIONAL([HAVE_PYTHON], [false])
+AM_CONDITIONAL([HAVE_GNU_GETTEXT_TRUE], [false])
+AM_CONDITIONAL([HAVE_GNU_GETTEXT], [false])
+AM_CONDITIONAL([HAVE_PHP], [false])
 
 dnl Bash completion.
 HEADING([Checking for bash completion])
@@ -203,8 +186,6 @@
                 [chmod +x,-w ocaml-dep.sh])
 AC_CONFIG_FILES([ocaml-link.sh],
                 [chmod +x,-w ocaml-link.sh])
-AC_CONFIG_FILES([php/extension/php-for-tests.sh],
-                [chmod +x,-w php/extension/php-for-tests.sh])
 AC_CONFIG_FILES([pick-guests.pl],
                 [chmod +x,-w pick-guests.pl])
 AC_CONFIG_FILES([podwrapper.pl],
@@ -227,16 +208,6 @@
                  cat/Makefile
                  common/errnostring/Makefile
                  common/edit/Makefile
-                 common/mlcustomize/Makefile
-                 common/mlgettext/Makefile
-                 common/mlpcre/Makefile
-                 common/mlprogress/Makefile
-                 common/mlstdutils/Makefile
-                 common/mlstdutils/guestfs_config.ml
-                 common/mltools/Makefile
-                 common/mlutils/Makefile
-                 common/mlvisit/Makefile
-                 common/mlxml/Makefile
                  common/options/Makefile
                  common/parallel/Makefile
                  common/progress/Makefile
@@ -248,110 +219,26 @@
                  common/windows/Makefile
                  config.sh
                  csharp/Makefile
-                 customize/Makefile
-                 daemon/daemon_config.ml
-                 daemon/Makefile
                  df/Makefile
                  dib/Makefile
                  diff/Makefile
-                 docs/Makefile
                  edit/Makefile
-                 erlang/Makefile
-                 erlang/examples/Makefile
-                 examples/Makefile
                  fish/Makefile
                  format/Makefile
                  fuse/Makefile
                  generator/Makefile
                  get-kernel/Makefile
                  gnulib/lib/Makefile
-                 gnulib/tests/Makefile
-                 gobject/libguestfs-gobject-1.0.pc
-                 gobject/Makefile
-                 golang/Makefile
-                 golang/examples/Makefile
-                 haskell/Makefile
                  include/Makefile
                  inspector/Makefile
-                 java/Makefile
-                 java/examples/Makefile
                  lib/Makefile
                  lib/libguestfs.pc
                  lib/local/libguestfs.pc
-                 lua/Makefile
-                 lua/examples/Makefile
                  make-fs/Makefile
-                 ocaml/META
-                 ocaml/Makefile
-                 ocaml/examples/Makefile
-                 perl/Build.PL
-                 perl/Makefile
-                 perl/examples/Makefile
-                 php/Makefile
-                 po-docs/Makefile
-                 po-docs/ja/Makefile
-                 po-docs/uk/Makefile
-                 po/Makefile
-                 python/Makefile
-                 python/examples/Makefile
-                 python/setup.py
-                 python/t/tests_helper.py
                  rescue/Makefile
                  resize/Makefile
-                 ruby/Makefile
-                 ruby/Rakefile
-                 ruby/examples/Makefile
-                 ruby/ext/guestfs/extconf.rb
-                 rust/Makefile
-                 rust/Cargo.toml
                  sparsify/Makefile
                  sysprep/Makefile
-                 test-data/Makefile
-                 test-data/binaries/Makefile
-                 test-data/blank-disks/Makefile
-                 test-data/files/Makefile
-                 test-data/phony-guests/Makefile
-                 test-data/phony-guests/guests.xml
-                 test-tool/Makefile
-                 tests/9p/Makefile
-                 tests/bigdirs/Makefile
-                 tests/btrfs/Makefile
-                 tests/c-api/Makefile
-                 tests/charsets/Makefile
-                 tests/create/Makefile
-                 tests/daemon/Makefile
-                 tests/daemon/captive-daemon.pm
-                 tests/discard/Makefile
-                 tests/disks/Makefile
-                 tests/disks/test-qemu-drive-libvirt.xml
-                 tests/disk-labels/Makefile
-                 tests/events/Makefile
-                 tests/gdisk/Makefile
-                 tests/hotplug/Makefile
-                 tests/http/Makefile
-                 tests/journal/Makefile
-                 tests/luks/Makefile
-                 tests/lvm/Makefile
-                 tests/md/Makefile
-                 tests/mount-local/Makefile
-                 tests/mountable/Makefile
-                 tests/nbd/Makefile
-                 tests/network/Makefile
-                 tests/ntfs/Makefile
-                 tests/parallel/Makefile
-                 tests/protocol/Makefile
-                 tests/qemu/Makefile
-                 tests/regressions/Makefile
-                 tests/relabel/Makefile
-                 tests/relative-paths/Makefile
-                 tests/rsync/Makefile
-                 tests/selinux/Makefile
-                 tests/syslinux/Makefile
-                 tests/tmpdirs/Makefile
-                 tests/tsk/Makefile
-                 tests/xfs/Makefile
-                 tests/xml/Makefile
-                 tests/yara/Makefile
                  tools/Makefile
                  website/index.html])
 
