--- ./inspector/Makefile.am	2021-03-11 17:20:47.000000000 +0100
+++ .//inspector/Makefile.am_new	2022-01-27 21:11:58.000000000 +0100
@@ -74,8 +74,8 @@
 	../gnulib/lib/libgnu.la
 
 # Manual pages and HTML files for the website.
-man_MANS = virt-inspector.1
-noinst_DATA = $(top_builddir)/website/virt-inspector.1.html
+man_MANS = 
+noinst_DATA = 
 
 virt-inspector.1 $(top_builddir)/website/virt-inspector.1.html: stamp-virt-inspector.pod
 
