--- .//edit/Makefile.am	2021-03-11 17:20:47.000000000 +0100
+++ .//edit/Makefile.am_new	2022-01-27 21:09:10.000000000 +0100
@@ -54,8 +54,8 @@
 	../gnulib/lib/libgnu.la
 
 # Manual pages and HTML files for the website.
-man_MANS = virt-edit.1
-noinst_DATA = $(top_builddir)/website/virt-edit.1.html
+man_MANS = 
+noinst_DATA = 
 
 virt-edit.1 $(top_builddir)/website/virt-edit.1.html: stamp-virt-edit.pod
 
