--- lib/Makefile.am	2021-11-26 10:49:57.000000000 +0100
+++ lib/Makefile.am_new	2022-01-24 22:25:10.000000000 +0100
@@ -96,11 +96,8 @@
 	journal.c \
 	launch.c \
 	launch-direct.c \
-	launch-libvirt.c \
 	launch-uml.c \
 	launch-unix.c \
-	libvirt-auth.c \
-	libvirt-domain.c \
 	lpj.c \
 	match.c \
 	mountable.c \
@@ -170,24 +167,7 @@
 libguestfs_la_LIBADD += $(FUSE_LIBS)
 endif
 
-if HAVE_LIBVIRT
-# Small utility to check for a needed libvirt version;
-# to be used in shell/script-based tests.
-noinst_PROGRAMS += libvirt-is-version
 
-libvirt_is_version_SOURCES = libvirt-is-version.c
-
-libvirt_is_version_LDADD = \
-	$(LIBVIRT_LIBS) \
-	$(LTLIBINTL)
-
-libvirt_is_version_CPPFLAGS = \
-	-DLOCALEBASEDIR=\""$(datadir)/locale"\"
-
-libvirt_is_version_CFLAGS = \
-	$(WARN_CFLAGS) $(WERROR_CFLAGS) \
-	$(LIBVIRT_CFLAGS)
-endif
 
 # Tests: main tests are in tests/c-api.  Here we just have some
 # internal tests of utility functions.
