--- .//lib/inspect-apps.c	2021-11-26 10:49:57.000000000 +0100
+++ .//lib/inspect-apps.c_new	2022-01-23 20:19:48.000000000 +0100
@@ -590,12 +590,6 @@
   if (!software_hive)
     return NULL;
 
-  if (guestfs_hivex_open (g, software_hive,
-                          GUESTFS_HIVEX_OPEN_VERBOSE, g->verbose,
-                          GUESTFS_HIVEX_OPEN_UNSAFE, 1,
-                          -1) == -1)
-    return NULL;
-
   /* Allocate apps list. */
   ret = safe_malloc (g, sizeof *ret);
   ret->len = 0;
@@ -615,7 +609,6 @@
   list_applications_windows_from_path (g, ret, hivepath2,
                                        sizeof hivepath2 / sizeof hivepath2[0]);
 
-  guestfs_hivex_close (g);
   return ret;
 }
 
@@ -624,70 +617,6 @@
                                      struct guestfs_application2_list *apps,
                                      const char **path, size_t path_len)
 {
-  CLEANUP_FREE_HIVEX_NODE_LIST struct guestfs_hivex_node_list *children = NULL;
-  int64_t node;
-  size_t i;
-
-  node = guestfs_hivex_root (g);
-
-  for (i = 0; node != 0 && i < path_len; ++i)
-    node = guestfs_hivex_node_get_child (g, node, path[i]);
-
-  if (node == 0)
-    return;
-
-  children = guestfs_hivex_node_children (g, node);
-  if (children == NULL)
-    return;
-
-  /* Consider any child node that has a DisplayName key.
-   * See also:
-   * http://nsis.sourceforge.net/Add_uninstall_information_to_Add/Remove_Programs#Optional_values
-   */
-  for (i = 0; i < children->len; ++i) {
-    const int64_t child = children->val[i].hivex_node_h;
-    int64_t value;
-    CLEANUP_FREE char *name = NULL, *display_name = NULL, *version = NULL,
-      *install_path = NULL, *publisher = NULL, *url = NULL, *comments = NULL;
-
-    /* Use the node name as a proxy for the package name in Linux.  The
-     * display name is not language-independent, so it cannot be used.
-     */
-    name = guestfs_hivex_node_name (g, child);
-    if (name == NULL)
-      continue;
-
-    value = guestfs_hivex_node_get_value (g, child, "DisplayName");
-    if (value) {
-      display_name = guestfs_hivex_value_string (g, value);
-      if (display_name) {
-        value = guestfs_hivex_node_get_value (g, child, "DisplayVersion");
-        if (value)
-          version = guestfs_hivex_value_string (g, value);
-        value = guestfs_hivex_node_get_value (g, child, "InstallLocation");
-        if (value)
-          install_path = guestfs_hivex_value_string (g, value);
-        value = guestfs_hivex_node_get_value (g, child, "Publisher");
-        if (value)
-          publisher = guestfs_hivex_value_string (g, value);
-        value = guestfs_hivex_node_get_value (g, child, "URLInfoAbout");
-        if (value)
-          url = guestfs_hivex_value_string (g, value);
-        value = guestfs_hivex_node_get_value (g, child, "Comments");
-        if (value)
-          comments = guestfs_hivex_value_string (g, value);
-
-        add_application (g, apps, name, display_name, 0,
-                         version ? : "",
-                         "", "",
-                         install_path ? : "",
-                         publisher ? : "",
-                         url ? : "",
-                         "", "",
-                         comments ? : "");
-      }
-    }
-  }
 }
 
 static void
