--- m4/guestfs-ocaml.m4	2022-01-23 08:12:00.000000000 +0100
+++ guestfs-ocaml.m4_new	2022-01-23 08:29:14.000000000 +0100
@@ -97,61 +97,6 @@
 fi
 AC_SUBST([OCAMLDEP_ONE_LINE])
 
-have_Hivex_OPEN_UNSAFE=no
-if test "x$enable_daemon" = "xyes"; then
-    OCAML_PKG_hivex=no
-    AC_CHECK_OCAML_PKG(hivex)
-    if test "x$OCAML_PKG_hivex" = "xno"; then
-        AC_MSG_ERROR([the OCaml module 'hivex' is required])
-    fi
-
-    # Check if Hivex has 'OPEN_UNSAFE' flag.
-    AC_MSG_CHECKING([for Hivex.OPEN_UNSAFE])
-    rm -f conftest.ml
-    echo 'let s = Hivex.OPEN_UNSAFE' > conftest.ml
-    if $OCAMLFIND ocamlc -package hivex -c conftest.ml >&5 2>&5 ; then
-        AC_MSG_RESULT([yes])
-        have_Hivex_OPEN_UNSAFE=yes
-    else
-        AC_MSG_RESULT([no])
-        have_Hivex_OPEN_UNSAFE=no
-    fi
-
-    dnl Check which OCaml runtime to link the daemon again.
-    dnl We can't use AC_CHECK_LIB here unfortunately because
-    dnl the other symbols are resolved by OCaml itself.
-    AC_MSG_CHECKING([which OCaml runtime we should link the daemon with])
-    if test "x$OCAMLOPT" != "xno"; then
-        for f in asmrun_pic asmrun; do
-            if test -f "$OCAMLLIB/lib$f.a"; then
-                CAMLRUN=$f
-                break
-            fi
-        done
-    else
-        for f in camlrun; do
-            if test -f "$OCAMLLIB/lib$f.a"; then
-                CAMLRUN=$f
-                break
-            fi
-        done
-    fi
-    if test "x$CAMLRUN" != "x"; then
-        AC_MSG_RESULT([$CAMLRUN])
-    else
-        AC_MSG_ERROR([could not find or link to libasmrun or libcamlrun])
-    fi
-    AC_SUBST([CAMLRUN])
-fi
-
-dnl Define HIVEX_OPEN_UNSAFE_FLAG based on test above.
-AS_IF([test "x$have_Hivex_OPEN_UNSAFE" = "xno"],[
-    HIVEX_OPEN_UNSAFE_FLAG="None"
-],[
-    HIVEX_OPEN_UNSAFE_FLAG="Some Hivex.OPEN_UNSAFE"
-])
-AC_SUBST([HIVEX_OPEN_UNSAFE_FLAG])
-
 OCAML_PKG_gettext=no
 OCAML_PKG_ounit2=no
 ounit_is_v2=no
