--- m4/guestfs-daemon.m4	2022-01-23 08:12:00.000000000 +0100
+++ guestfs-daemon.m4_new	2022-01-23 08:31:20.000000000 +0100
@@ -93,15 +93,6 @@
     ], [])
 ],[AC_MSG_WARN([Linux capabilities library (libcap) not found])])
 
-dnl hivex library (required)
-PKG_CHECK_MODULES([HIVEX], [hivex],[
-    AC_SUBST([HIVEX_CFLAGS])
-    AC_SUBST([HIVEX_LIBS])
-    AC_DEFINE([HAVE_HIVEX],[1],[hivex library found at compile time.])
-],
-    [AC_MSG_FAILURE([hivex library is required])])
-AM_CONDITIONAL([HAVE_HIVEX],[test "x$HIVEX_LIBS" != "x"])
-
 dnl librpm library (optional)
 PKG_CHECK_MODULES([LIBRPM], [rpm >= 4.6.0],[
     AC_SUBST([LIBRPM_CFLAGS])
