--- configure.ac	2021-12-24 10:45:54.000000000 +0100
+++ configure.ac_new	2022-01-24 20:21:20.000000000 +0100
@@ -198,8 +198,6 @@
                 [chmod +x,-w ocaml-dep.sh])
 AC_CONFIG_FILES([ocaml-link.sh],
                 [chmod +x,-w ocaml-link.sh])
-AC_CONFIG_FILES([php/extension/php-for-tests.sh],
-                [chmod +x,-w php/extension/php-for-tests.sh])
 AC_CONFIG_FILES([podwrapper.pl],
                 [chmod +x,-w podwrapper.pl])
 AC_CONFIG_FILES([run],
@@ -227,61 +225,17 @@
                  common/windows/Makefile
                  config.sh
                  csharp/Makefile
-                 daemon/daemon_config.ml
-                 daemon/Makefile
-                 docs/Makefile
-                 erlang/Makefile
-                 erlang/examples/Makefile
-                 examples/Makefile
                  fish/Makefile
                  fuse/Makefile
                  generator/Makefile
                  gnulib/lib/Makefile
                  gobject/libguestfs-gobject-1.0.pc
                  gobject/Makefile
-                 golang/Makefile
-                 golang/examples/Makefile
-                 haskell/Makefile
                  include/Makefile
-                 java/Makefile
-                 java/examples/Makefile
                  lib/Makefile
                  lib/libguestfs.pc
                  lib/local/libguestfs.pc
-                 lua/Makefile
-                 lua/examples/Makefile
-                 ocaml/META
-                 ocaml/Makefile
-                 ocaml/examples/Makefile
-                 perl/Build.PL
-                 perl/Makefile
-                 perl/examples/Makefile
-                 php/Makefile
-                 po-docs/Makefile
-                 po-docs/ja/Makefile
-                 po-docs/uk/Makefile
-                 po/Makefile
-                 python/Makefile
-                 python/examples/Makefile
-                 python/setup.py
-                 python/t/tests_helper.py
                  rescue/Makefile
-                 ruby/Makefile
-                 ruby/Rakefile
-                 ruby/examples/Makefile
-                 ruby/ext/guestfs/extconf.rb
-                 rust/Makefile
-                 rust/Cargo.toml
-                 test-data/Makefile
-                 test-data/binaries/Makefile
-                 test-data/blank-disks/Makefile
-                 test-data/files/Makefile
-                 test-data/phony-guests/Makefile
-                 test-data/phony-guests/guests.xml
-                 test-tool/Makefile
-                 tests/Makefile
-                 tests/daemon/captive-daemon.pm
-                 tests/disks/test-qemu-drive-libvirt.xml
                  website/index.html])
 
 AC_OUTPUT
