--- Makefile.in	2021-12-24 10:46:16.000000000 +0100
+++ Makefile.in_new	2022-01-24 19:25:24.000000000 +0100
@@ -453,14 +453,12 @@
 	$(TEST_LOG_FLAGS)
 DIST_SUBDIRS = common/mlstdutils generator test-data gnulib/lib \
 	common/errnostring common/protocol common/qemuopts \
-	common/utils common/structs include lib docs examples \
+	common/utils common/structs include lib docs  \
 	common/mlutils bundled/ocaml-augeas common/mlpcre \
 	common/mlgettext daemon appliance tests common/edit \
 	common/options common/parallel common/progress common/visit \
-	common/windows test-tool fish rescue bash perl perl/examples \
-	ocaml ocaml/examples python python/examples ruby ruby/examples \
-	java java/examples haskell php erlang erlang/examples lua \
-	lua/examples gobject golang golang/examples rust csharp fuse \
+	common/windows fish rescue bash \
+	ocaml gobject csharp fuse \
 	po po-docs
 am__DIST_COMMON = $(srcdir)/Makefile.in $(srcdir)/config.h.in \
 	$(srcdir)/config.sh.in $(srcdir)/installcheck.sh.in \
@@ -895,11 +893,11 @@
 # After all source files were used we can generate the translation strings
 SUBDIRS = common/mlstdutils generator test-data gnulib/lib \
 	common/errnostring common/protocol common/qemuopts \
-	common/utils common/structs include lib docs examples \
+	common/utils common/structs include lib docs \
 	common/mlutils bundled/ocaml-augeas common/mlpcre \
 	common/mlgettext $(am__append_1) $(am__append_2) tests \
 	common/edit common/options common/parallel common/progress \
-	common/visit common/windows test-tool fish rescue bash \
+	common/visit common/windows fish rescue bash \
 	$(am__append_3) $(am__append_4) $(am__append_5) \
 	$(am__append_6) $(am__append_7) $(am__append_8) \
 	$(am__append_9) $(am__append_10) $(am__append_11) \
@@ -1712,32 +1710,6 @@
 
 dist-hook: docs/C_SOURCE_FILES po/POTFILES po/POTFILES-ml
 
-# This has to be in the top-level Makefile.am so that we have access
-# to DIST_SUBDIRS.
-docs/C_SOURCE_FILES: configure.ac
-	rm -f $@ $@-t
-	find $(DIST_SUBDIRS) -name '*.[ch]' | \
-	grep -v -E '^(builder/index-parse\.|builder/index-scan\.|examples/|gobject/|java/com_redhat_et_libguestfs|perl/|php/extension/config\.h|ruby/ext/guestfs/extconf\.h|tests/|test-data/|bundled/)' | \
-	grep -v -E '/(guestfs|rc)_protocol\.' | \
-	grep -v -E '.*/errnostring\.' | \
-	grep -v -E '.*-gperf\.' | \
-	grep -v -E '/dummy.c$$' | \
-	LC_ALL=C sort -u > $@-t
-	mv $@-t $@
-
-# For more information about translations, see po/Makefile.am.
-po/POTFILES: configure.ac
-	rm -f $@ $@-t
-	cd $(srcdir); \
-	find $(DIST_SUBDIRS) -name '*.c' | \
-	grep -v -E '^(examples|perl/(blib|examples)|po-docs|tests|test-data|bundled)/' | \
-	grep -v -E '/((guestfs|rc)_protocol\.c|dummy\.c)$$' | \
-	grep -v -E '^python/utils\.c$$' | \
-	grep -v -E '^perl/lib/Sys/Guestfs\.c$$' | \
-	grep -v -E '.*-(tests|gperf)\.c$$' | \
-	LC_ALL=C sort -u > $@-t
-	mv $@-t $@
-
 # Try to stop people using 'make install' without 'DESTDIR'.
 install:
 	@if test "x$(DESTDIR)" != "x" || test "x$(REALLY_INSTALL)" = "xyes"; \
@@ -1778,7 +1750,7 @@
 # is NOT a substitute for proper testing!
 
 quickcheck:
-	$(top_builddir)/run test-tool/libguestfs-test-tool $(QUICKCHECK_TEST_TOOL_ARGS)
+	$(top_builddir)/run $(QUICKCHECK_TEST_TOOL_ARGS)
 
 # Non-standard tests.
 
