--- m4/guestfs-daemon.m4	2020-03-06 20:31:08.000000000 +0100
+++ m4/guestfs-daemon.m4_new	2022-01-25 20:43:13.000000000 +0100
@@ -93,14 +93,6 @@
     ], [])
 ],[AC_MSG_WARN([Linux capabilities library (libcap) not found])])
 
-dnl hivex library (required)
-PKG_CHECK_MODULES([HIVEX], [hivex],[
-    AC_SUBST([HIVEX_CFLAGS])
-    AC_SUBST([HIVEX_LIBS])
-    AC_DEFINE([HAVE_HIVEX],[1],[hivex library found at compile time.])
-],
-    [AC_MSG_FAILURE([hivex library is required])])
-AM_CONDITIONAL([HAVE_HIVEX],[test "x$HIVEX_LIBS" != "x"])
 
 dnl systemd journal library (optional)
 PKG_CHECK_MODULES([SD_JOURNAL], [libsystemd],[
