--- examples/Makefile.am	2022-01-23 08:12:00.000000000 +0100
+++ Makefile.am_new	2022-01-23 08:56:42.000000000 +0100
@@ -33,9 +33,6 @@
 if HAVE_LIBVIRT
 noinst_PROGRAMS += copy-over libvirt-auth
 endif
-if HAVE_HIVEX
-noinst_PROGRAMS += virt-dhcp-address
-endif
 if HAVE_FUSE
 noinst_PROGRAMS += mount-local
 endif
@@ -116,17 +113,6 @@
 	$(top_builddir)/lib/libguestfs.la
 endif
 
-if HAVE_HIVEX
-virt_dhcp_address_SOURCES = virt-dhcp-address.c
-virt_dhcp_address_CPPFLAGS = \
-	-DGUESTFS_NO_DEPRECATED=1 \
-	-I$(top_srcdir)/include
-virt_dhcp_address_CFLAGS = \
-	$(WARN_CFLAGS) $(WERROR_CFLAGS)
-virt_dhcp_address_LDADD = \
-	$(top_builddir)/lib/libguestfs.la
-endif
-
 man_MANS = guestfs-examples.3
 noinst_DATA = $(top_builddir)/website/guestfs-examples.3.html
 
