--- generator/actions.ml	2022-01-23 08:12:00.000000000 +0100
+++ actions.ml_new	2022-01-23 11:16:15.000000000 +0100
@@ -49,8 +49,6 @@
   Actions_core.daemon_functions @
   Actions_core_deprecated.daemon_functions @
   Actions_debug.daemon_functions @
-  Actions_hivex.daemon_functions @
-  Actions_hivex_deprecated.daemon_functions @
   Actions_inspection.daemon_functions @
   Actions_inspection_deprecated.daemon_functions @
   Actions_tsk.daemon_functions @
