--- generator/Makefile.am	2022-01-23 08:12:00.000000000 +0100
+++ generator/Makefile.am_new	2022-01-23 08:17:05.000000000 +0100
@@ -29,10 +29,6 @@
 	actions_core_deprecated.mli \
 	actions_debug.ml \
 	actions_debug.mli \
-	actions_hivex.ml \
-	actions_hivex.mli \
-	actions_hivex_deprecated.ml \
-	actions_hivex_deprecated.mli \
 	actions_inspection.ml \
 	actions_inspection.mli \
 	actions_inspection_deprecated.ml \
@@ -130,8 +126,6 @@
 	actions_core.cmo \
 	actions_core_deprecated.cmo \
 	actions_debug.cmo \
-	actions_hivex.cmo \
-	actions_hivex_deprecated.cmo \
 	actions_inspection.cmo \
 	actions_inspection_deprecated.cmo \
 	actions_internal_tests.cmo \
