--- installcheck.sh.in	2021-11-26 10:49:57.000000000 +0100
+++ installcheck.sh.in_new	2022-01-24 08:00:24.000000000 +0100
@@ -41,7 +41,6 @@
 cp @libdir@/libguestfs.so.0.* lib/.libs/
 
 # Copy installed binaries into the right places.
-cp @bindir@/libguestfs-test-tool  test-tool/
 cp @bindir@/guestfish             fish/
 cp @bindir@/guestmount            fuse/
 cp @bindir@/virt-alignment-scan   align/
