--- Makefile.am	2021-11-26 10:49:56.000000000 +0100
+++ Makefile.am_new	2022-01-24 20:04:18.000000000 +0100
@@ -24,30 +24,23 @@
 
 # Files and other test data used by the tests.  Must be before any
 # tests run.
-SUBDIRS += test-data
 
 # Basic source for the library.
 SUBDIRS += gnulib/lib
 SUBDIRS += common/errnostring common/protocol common/qemuopts
 SUBDIRS += common/utils
 SUBDIRS += common/structs
-SUBDIRS += include lib docs examples
+SUBDIRS += include lib
 
 # The daemon and the appliance.
 SUBDIRS += common/mlutils
 SUBDIRS += bundled/ocaml-augeas
 SUBDIRS += common/mlpcre
 SUBDIRS += common/mlgettext
-if ENABLE_DAEMON
-SUBDIRS += daemon
-endif
 if ENABLE_APPLIANCE
 SUBDIRS += appliance
 endif
 
-# Tests.
-SUBDIRS += tests
-
 # Common code used by the tools.
 SUBDIRS += common/edit
 SUBDIRS += common/options
@@ -57,48 +50,14 @@
 SUBDIRS += common/windows
 
 # Small tools written in C.
-SUBDIRS += test-tool fish rescue
+SUBDIRS += fish rescue
 
 # bash-completion
 SUBDIRS += bash
 
-# Language bindings.
-if HAVE_PERL
-SUBDIRS += perl perl/examples
-endif
-if HAVE_OCAML
-SUBDIRS += ocaml ocaml/examples
-endif
-if HAVE_PYTHON
-SUBDIRS += python python/examples
-endif
-if HAVE_RUBY
-SUBDIRS += ruby ruby/examples
-endif
-if HAVE_JAVA
-SUBDIRS += java java/examples
-endif
-if HAVE_HASKELL
-SUBDIRS += haskell
-endif
-if HAVE_PHP
-SUBDIRS += php
-endif
-if HAVE_ERLANG
-SUBDIRS += erlang erlang/examples
-endif
-if HAVE_LUA
-SUBDIRS += lua lua/examples
-endif
 if HAVE_GOBJECT
 SUBDIRS += gobject
 endif
-if HAVE_GOLANG
-SUBDIRS += golang golang/examples
-endif
-if HAVE_RUST
-SUBDIRS += rust
-endif
 
 # Unconditional because nothing is built yet.
 SUBDIRS += csharp
@@ -111,18 +70,12 @@
 # After all source files were used we can generate the translation strings
 SUBDIRS += po
 
-# po-docs must come after tools, inspector.
-if HAVE_PO4A
-SUBDIRS += po-docs
-endif
-
 EXTRA_DIST = \
 	AUTHORS HACKING RELEASES TODO \
 	.gitignore \
 	.lvimrc \
 	.mailmap \
 	bugs-in-changelog.sh \
-	bindtests \
 	build-aux/config.rpath \
 	check-mli.sh \
 	common/.gitignore \
@@ -157,7 +110,6 @@
 	logo/fish-10yrs.svg logo/fish-10yrs.png \
 	logo/virt-builder.svg \
 	podcheck.pl \
-	tests/test-functions.sh \
 	tmp/.gitignore \
 	valgrind-suppressions \
 	website/bugs.png \
@@ -213,30 +165,6 @@
 
 # This has to be in the top-level Makefile.am so that we have access
 # to DIST_SUBDIRS.
-docs/C_SOURCE_FILES: configure.ac
-	rm -f $@ $@-t
-	find $(DIST_SUBDIRS) -name '*.[ch]' | \
-	grep -v -E '^(builder/index-parse\.|builder/index-scan\.|examples/|gobject/|java/com_redhat_et_libguestfs|perl/|php/extension/config\.h|ruby/ext/guestfs/extconf\.h|tests/|test-data/|bundled/)' | \
-	grep -v -E '/(guestfs|rc)_protocol\.' | \
-	grep -v -E '.*/errnostring\.' | \
-	grep -v -E '.*-gperf\.' | \
-	grep -v -E '/dummy.c$$' | \
-	LC_ALL=C sort -u > $@-t
-	mv $@-t $@
-
-# For more information about translations, see po/Makefile.am.
-po/POTFILES: configure.ac
-	rm -f $@ $@-t
-	cd $(srcdir); \
-	find $(DIST_SUBDIRS) -name '*.c' | \
-	grep -v -E '^(examples|perl/(blib|examples)|po-docs|tests|test-data|bundled)/' | \
-	grep -v -E '/((guestfs|rc)_protocol\.c|dummy\.c)$$' | \
-	grep -v -E '^python/utils\.c$$' | \
-	grep -v -E '^perl/lib/Sys/Guestfs\.c$$' | \
-	grep -v -E '.*-(tests|gperf)\.c$$' | \
-	LC_ALL=C sort -u > $@-t
-	mv $@-t $@
-
 # Try to stop people using 'make install' without 'DESTDIR'.
 install:
 	@if test "x$(DESTDIR)" != "x" || test "x$(REALLY_INSTALL)" = "xyes"; \
@@ -292,7 +220,7 @@
 # is NOT a substitute for proper testing!
 
 quickcheck:
-	$(top_builddir)/run test-tool/libguestfs-test-tool $(QUICKCHECK_TEST_TOOL_ARGS)
+	$(top_builddir)/run  $(QUICKCHECK_TEST_TOOL_ARGS)
 
 # Non-standard tests.
 
@@ -353,11 +281,6 @@
 	SKIP_TEST_RHBZ914931=1 \
 	SKIP_TEST_INSPECT_FSTAB_MD_SH=1
 
-check-uml:
-	$(MAKE) LIBGUESTFS_BACKEND=uml $(SKIP_TESTS_FAILING_IN_UML) check
-
-check-valgrind-uml:
-	$(MAKE) LIBGUESTFS_BACKEND=uml $(SKIP_TESTS_FAILING_IN_UML) check-valgrind
 
 QEMUDIR = $(HOME)/d/qemu
 QEMUBINARY = $(QEMUDIR)/x86_64-softmmu/qemu-system-x86_64
