--- lib/Makefile.am	2022-01-24 23:37:38.000000000 +0100
+++ lib/Makefile.am_new	2022-01-25 21:53:45.000000000 +0100
@@ -58,6 +58,9 @@
 	../common/utils/guestfs-utils.h \
 	../common/structs/structs-cleanups.h \
 	../include/guestfs.h \
+        ../gnulib/lib/error.h \
+        ../gnulib/lib/gettext.h \
+	/opt/local/include/libintl.h \
 	guestfs-internal.h \
 	guestfs-internal-all.h \
 	actions-0.c \
